torch/_dynamo/decorators.py

```diff
@@ -114,7 +114,11 @@ def __init__(
         skip_guard_eval_unsafe: bool = False,
         force_backend=None,
     ) -> None:
-        if force_backend is not None and stance != "default":
+        if (
+            stance != "eager_then_compile"
+            and force_backend is not None
+            and stance != "default"
+        ):
             raise RuntimeError("non-default stance cannot have force_backend set")
```

Comment:
Update this error message? (also could cleanup the above condition to `force_backend is not None and stance not in (...)`)

Reply:
Whoops I meant to remove this code